apps/omg_watcher/test/fixtures.exs

```diff
@@ -114,6 +114,8 @@ defmodule OMG.Watcher.Fixtures do
   deffixture watcher(db_initialized, root_chain_contract_config) do
     :ok = root_chain_contract_config
     :ok = db_initialized
+
+    DeferredConfig.populate(:omg_watcher)
```

Comment:
is this necessary? we're starting the watcher right next there, and that start would do the `populate`